mmedit/models/backbones/encoder_decoders/decoders/fba_decoder.py

```diff
@@ -0,0 +1,119 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvWS2d
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class FBADecoder(nn.Module):
+
+    def __init__(self,
+                 batch_norm=False,
+                 norm_cfg=None,
+                 act_cfg=dict(type='ReLU')):
+        super(FBADecoder, self).__init__()
+
+        # Pyramid Pooling Module
+        pool_scales = (1, 2, 3, 6)
+        self.batch_norm = batch_norm
+
+        self.ppm = []
+
+        for scale in pool_scales:
+            self.ppm.append(
+                nn.Sequential(
+                    nn.AdaptiveAvgPool2d(scale),
+                    ConvWS2d(2048, 256, kernel_size=1, bias=True),
+                    norm(256, self.batch_norm), nn.LeakyReLU()))
```

Comment:
Why norm_cfg and act_cfg are needed if all are hard-coded as batch_norm and LeakyReLU